client/desktop/lib/updater/index.js

```diff
@@ -59,18 +61,18 @@ class Updater extends EventEmitter {
 		if ( ! this._hasPrompted ) {
 			this._hasPrompted = true;
 
-			dialog.showMessageBox( updateDialogOptions, ( button ) => {
-				this._hasPrompted = false;
+			const selected = await dialog.showMessageBox( mainWindow, updateDialogOptions );
+			const button = selected.response;
 
-				if ( button === 0 ) {
-					// Confirm
-					this.onConfirm();
-				} else {
-					this.onCancel();
-				}
+			if ( button === 0 ) {
+				// Confirm
```

Comment:
Nitpick. Is this comment necessary? The line below, `this.onConfirm();` already tells that this is the confirmation branch well.

Reply:
You're totally right - I copied the changes verbatim but will remove the unnecessary comment prior to landing. Thanks!